### ISSUE TITLE
Permission denied on socket with 5.0.8

### ISSUE THREAD

**Issue by JasonBarnabe** [phusion/passenger#1520]
Upon upgrading my Rails site from Passenger 4.0.53 to 5.0.8, I'm greeted with a 502 error. Error log says:

> 2015/05/24 02:36:43 [crit] 20408#0: *179 connect() to unix:/tmp/passenger.FD57G8h/agents.s/server failed (13: Permission denied) while connecting to upstream, client: 1.2.3.4, server: userstyles.org, request: "GET / HTTP/1.1", upstream: "passenger:unix:/tmp/passenger.FD57G8h/agents.s/server:", host: "userstyles.org"

I have the following settings which may be relevant.
```
    passenger_user_switching off;
    passenger_default_user www;
    passenger_default_group www;
```
`www` is the user that started nginx (with sudo). `www` does not own all the files of the Rails app, but can `r` and `x` them.

Downgrading to 4.0.53, reinstalling the nginx module, and restarting brings things back to life.

**Comment by FooBarWidget** [comment]
Which OS are you on?

**Comment by JasonBarnabe** [comment]
CentOS 6.6.

**Comment by FooBarWidget** [comment]
This bug has been confirmed and fixed. The fix will be part of 5.0.9. Thank you for reporting this.
